packages/styled-icons/generate/generate.js

```diff
@@ -126,11 +126,6 @@ const generate = async () => {
       }
     }
 
-    // If the base icon has a fill of `none`, replace it
-    if (icon.attrs.fill === 'none') {
```

Comment:
I can optionally, re-add this to where it does a check that `stroke !== 'currentColor'` again if you want @jacobwgillespie. However, it does work without it. The issue wasn't that `fill` was `none` but that I had stripped out the `stroke="currentColor"`

Reply:
Cool, yeah this was my attempt to fix this issue that only half worked, I'm cool with removing the partial measure!